app/jobs/concerns/maintenance_tasks/task_job_concern.rb

```diff
@@ -0,0 +1,146 @@
+# frozen_string_literal: true
+module MaintenanceTasks
+  # Concern that holds the behaviour of the job that runs the tasks. It is
+  # included in {TaskJob} and if {MaintenanceTasks.job MaintenanceTasks.job} is
```

Comment:
Yeah the thing is that it generates a link with only `job` by default:
<img width="530" alt="image" src="https://user-images.githubusercontent.com/3535/117320124-edceaf80-ae59-11eb-98a8-2860d6315b70.png">

But actually right now there's no link because we haven't properly documented the attributes on MaintenanceTasks. I'll remove the link in the meantime.

Reply:
Ah my bad, didn't realize it wouldn't generate the link properly without that and assumed this was a typo 😂